PyMPDATA/solver.py

```diff
@@ -107,8 +109,12 @@ def advector(self) -> VectorField:
 
     @property
     def g_factor(self) -> ScalarField:
-        """g_factor field (with halo), unmodified by advance(),
-        assumed to be constant-in-time"""
+        """G_factor field (with halo), unmodified by advance(), assumed to be constant-in-time.
+        Can be used i.e. to transform the geometry of the advector
+        and advectee into eg a spherical one,
+        or to account for the grid spacing in the advection equation,
```

Comment:
not sure if I understand "grid spacing" here?

Reply:
I've deleted this line.